recipes/r-xmsannotator/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "xMSannotator" %}
+{% set version = "1.3.2+recetox0" %}
+
+package:
+  name: "r-{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/RECETOX/xMSannotator/archive/packaging.tar.gz
+  sha256: 22ec4034585ecde641f531693acaab4145617f9b1cd7787eda0c71e4cda961ea
+
+build:
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/
+    - lib/R/lib/
+
+
+requirements:
+  build:
+    - r-base >=4
+    - r-dplyr
+    - r-flashclust
+    - r-wgcna
+    - r-tidyr
+    - r-purrr
+    - r-tibble
+
+  run:
+    - r-base >=4
+    - r-dplyr >=1
+    - r-flashclust
+    - r-wgcna
+    - r-tidyr
+    - r-purrr
+    - r-tibble
+    - r-data.table
+    - bioconductor-rhdf5
+    - r-rcpp
+    - r-arrow
+
+  test:
+    commands:
+      - $Rscript -e "library('xmsannotator')"
```

This block should be at root level.

```suggestion
test:
  commands:
    - $Rscript -e "library('xmsannotator')"
```

---

Thank you, I couldn't wrap my head around what was the problem.